config/taxonomy/objects/policy_manager_request.json

```diff
@@ -0,0 +1,74 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Policy Manager Request",
+	"type": "object",
+	"properties": {
+	  "context": {
+		"$ref": "taxonomy.json#/definitions/PolicyManagerRequestContext"
+	  },
+	  "action": {
+		"$ref": "#/definitions/PolicyManagerRequestAction"
+	  },
+	  "resource": {
+		"$ref": "#/definitions/Resource"
+	  }
+	},
+	"additionalProperties": false,
+	"required": [
+	  "action",
+	  "resource"
+	],
+	"definitions": {
+	  "PolicyManagerRequestAction": {
+		"type": "object",
+		"properties": {
+		  "actionType": {
+			"$ref": "#/definitions/ActionType"
+		  },
+		  "processingLocation": {
+			"$ref": "taxonomy.json#/definitions/ProcessingLocation"
+		  }
+		}
+	  },
+	  "ActionType": {
+		"type": "string",
+		"enum": [
+		  "read",
```

Comment:
actions should also be in a layer?

Reply:
We do not currently see it as something configurable by any component.

Reply:
Eventually this will be used by M4Dapplication right?